discord/ext/pages/pagination.py

```diff
@@ -174,14 +174,21 @@ def update_files(self) -> list[discord.File] | None:
         """
         for file in self._files:
             if not isinstance(file.fp, BufferedReader):
```

Comment:
This should be inverted
```suggestion
            if isinstance(file.fp, BufferedReader):
```

Reply:
That is incorrect. See #1869

Reply:
No it's correct because you changed the way it works

Reply:
oh wait nvm

Reply:
This is the way I have it in my code and it works perfectly. Perhaps it can work both ways - can't check now, gotta go.

Reply:
> That is incorrect. See #1869

With the state it is currently, the below code will run if `file.fp` is a `BufferedReader`, causing the exception in #1869 because it essentially would revert that change.
Have you actually tested your changes?

Reply:
> > That is incorrect. See #1869
> 
> With the state it is currently, the below code will run if `file.fp` is a `BufferedReader`, causing the exception in #1869 because it essentially would revert that change.
> 
> https://github.com/CompuGenius/pycord/blob/d7eaa0200dfba7c1a0c114ba9e70379fd508668d/discord/ext/pages/pagination.py#L185-L190
> 
> Have you actually tested your changes?

As I said in the PR description, and I also sent a reply comment (deleted when @Lulalaby originally agreed) - yes, I did test them. This is what I currently have that works. Perhaps test it yourself, maybe it's a "works on my machine" thing.

Reply:
Look at [this](https://github.com/Pycord-Development/pycord/pull/1869/commits/17c937e828bbea97851405972e75ebb5a886b91e). I am correct. We only want the change if it's `io.BytesIO`, NOT `io.BufferedReader`.

Reply:
You shouldn't delete a comment just bcs I agree 😅 I'm just responsible for everything around pycord and api review.

Reply:
> Look at [this](https://github.com/Pycord-Development/pycord/pull/1869/commits/17c937e828bbea97851405972e75ebb5a886b91e). I am correct. We only want the change if it's `io.BytesIO`, NOT `io.BufferedReader`.

Apologies for the confusion, I was misreading the diff in this file

Reply:
> > Look at [this](https://github.com/Pycord-Development/pycord/pull/1869/commits/17c937e828bbea97851405972e75ebb5a886b91e). I am correct. We only want the change if it's `io.BytesIO`, NOT `io.BufferedReader`.
> 
> Apologies for the confusion, I was misreading the diff in this file

No problem ☺️